backend/src/main/java/com/openelements/maven/initializer/backend/service/MavenBom.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.openelements.maven.initializer.backend.service;
+
+/** Represents a Maven BOM (Bill of Materials) for dependency management. */
+public record MavenBom(String groupId, String artifactId, String version) {}
```

Comment:
Can be merge with MavenDependency. A BOM is also a dependency with different scope and type. That is set during runtime.